tools/internal_ci/linux/grpc_distribtests_python.sh

```diff
@@ -68,12 +68,11 @@ cp -r artifacts/* input_artifacts/ || true
 # Run all python linux distribtests
 # We run the distribtests even if some of the artifacts have failed to build, since that gives
 # a better signal about which distribtest are affected by the currently broken artifact builds.
-if [[ "${IS_AARCH64_MUSL}" == "True" ]]; then
-  # We're using alpine as tag in distribtest targets.
-  tools/run_tests/task_runner.py -f distribtest linux python aarch64 alpine -j 12 -x distribtests/sponge_log.xml || FAILED="true"
-else
-  tools/run_tests/task_runner.py -f distribtest linux python ${TASK_RUNNER_EXTRA_FILTERS} -j 12 -x distribtests/sponge_log.xml || FAILED="true"
-fi
+
+# We're using alpine as tag in distribtest targets for musllinux_1_1 artifacts, so exclude filters must use this tag 
+DISTRIB_TASK_RUNNER_EXTRA_FILTERS="${TASK_RUNNER_EXTRA_FILTERS//musllinux_1_1/alpine}"
```

Comment:
Note for future: we should find a better way to do it. 

We also should add a comment to show how the variable looks like: `-e aarch64 musllinux_1_1`
Just so we understand what actually gets edited. Last time we deleted the `platform`, but it was a part of `-f` flag args, not `-e`. 
Without it, it's not clear what's happening.